applications/accounts/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/keycloak/keycloak:26.3.5
+FROM quay.io/keycloak/keycloak:26.4
```

Comment:
This automatically updates to the latest minor version. It could lead to having different versions across envs for the same app.

Reply:
It should be no harm to use the latest patch update for keycloak, they respect semver quite well and release quite often